Collection Interfaces/List Interface/boolean isEmpty()/isEmptyFunc.java

```diff
@@ -0,0 +1,22 @@
+import java.util.*; 
```

Comment:
add documentation(in form of comment explaining [ isEmpty() method ] in 2-3 lines.

Reply:
@jakansha2001  check once the comments i added in first line. From next time try include it too in the code.

Reply:
@JyotiJauhari Thank you for this info. I will take care of this thing from now onwards.